src/Components/WebAssembly/Sdk/src/targets/Microsoft.NET.Sdk.BlazorWebAssembly.Current.targets

```diff
@@ -32,7 +32,7 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <PropertyGroup>
     <!-- Paths to tools, tasks, and extensions are calculated relative to the BlazorWebAssemblySdkDirectoryRoot. This can be modified to test a local build. -->
-    <BlazorWebAssemblySdkDirectoryRoot Condition="'$(BlazorWebAssemblySdkDirectoryRoot)'==''">$(MSBuildThisFileDirectory)..\..\</BlazorWebAssemblySdkDirectoryRoot>
+    <BlazorWebAssemblySdkDirectoryRoot Condition="'$(BlazorWebAssemblySdkDirectoryRoot)'==''">$(MSBuildThisFileDirectory)..\</BlazorWebAssemblySdkDirectoryRoot>
```

Comment:
Our integration tests run from the build output and do not exercise this code path. Our templates will once we have a .NET SDK that includes the Blazor SDK

Reply:
>  Our templates will once we have a .NET SDK that includes the Blazor SDK

When will we be doing this?

Reply:
We need to insert the Blazor SDK -> .NET SDK -> Installer -> update our global.json to use this build -> Golden!